FHEM/00_SIGNALduino.pm

```diff
@@ -1006,8 +1006,14 @@ sub SIGNALduino_parseResponse($$$) {
 sub SIGNALduino_ResetDevice($) {
 	my $hash = shift;
 	my $name = $hash->{NAME};
-	if (!defined($hash->{helper}{resetInProgress}))
-	{
+
+	if (InternalVal($name,"DeviceName","none") eq "none") { # for dummy device
```

Comment:
Vielleicht wäre es besser auf IsDummy die Prüfung zu machen, anstelle auf none im Gerätenamen.

Reply:
Kann man machen aber an dieser Stelle schlagen dann deine Test fail.

`test_reset_1-definition.txt`

Entweder die Position ist ungünstig oder die Tests sind ungünstig. Da fehlt mir gerade noch der Durchblick.

Reply:
Wenn wir die Prüfung mit isDummy() machen geht es auch aber sobald der User "spielt an den Attributen" und das Attribut dummy mit dem Wert 1 löscht, so geht der reset nicht mehr. Dem ganzen kann man vorbeugen wenn wir es auf den Gerätenamen none abfragen.

Reply:
Also die Tests welche mit einem Dummy betrieben werden schlagen fehl, weil nach der neuen Abfrage der Sprung zur näcshten Sub `SIGNALduino_Connect `unterbunden wird.

Rein von der Theorie, denke ich, das es so richtig wäre bei einem Dummy dort abzubrechen.
Bei einem richtigen Device müsste der Test natürlich weiter gehen aber wir haben zum testen virtuel kein richtiges Device. Kurzum, die Tests müsste man mal überdenken oder anders angehen.

Reply:
Ein Dummy ist meiner Meinung nach nur ein Dummy, wenn das Attribut dummy auf 1 gesetzt wird.
Alles andere ist in diesem Sinne kein "dummy".

Reply:
Dann sehen wir es als gegeben hin und beziehen uns auf isDummy. Wenn jemand manuell eingreift und das Attribut löscht, is es kein Dummy mehr. Ok.

Ich schaue mir die Tests dann nochmal an mit dem Wissen.